modules/fb_instant_articles_display/fb_instant_articles_display.module

```diff
@@ -13,15 +13,15 @@
 */
 function fb_instant_articles_display_init() {
  // Configure Instant Articles Transformer logger to append to Drupal log if enabled
- $enable_transformer_logging = variable_get('fb_instant_articles_display_enable_transformer_logging', FALSE);
+ $enable_transformer_logging = variable_get('fb_instant_articles_display_enable_transformer_logging');
```

Comment:
I think I would like the default to remain off, since the transformer logger is quite chatty on the DEBUG level (unless there's a Drupal convention to have modules enable as much logging as possible by default and let people dial it down if they want).